addons/hr_attendance/tests/test_hr_attendance_constraints.py

```diff
@@ -62,3 +62,17 @@ def test_new_attendances(self):
             self.open_attendance.write({
                 'check_out': time.strftime('%Y-%m-10 11:30'),
             })
+
+    def test_time_format_attendance(self):
+        self.env['res.lang']._activate_lang('en_USn')
```

Comment:
```suggestion
        self.env['res.lang']._activate_lang('en_US')
```
Isn't there a typo in the lang name here ?